### PR TITLE
libcryptsetup 0.6.0 changes

### PR DESCRIPTION
**Pull request by jbaublitz**
Related to stratis-storage/libcryptsetup-rs#228
Related https://github.com/stratis-storage/project/issues/507

### CODE REVIEW

**Comment by mulkieran** [comment]
Changes all look good.

**Review by bmr-cymru — APPROVED** [review]

File: .github/workflows/cargo.yml [hunk 1]
```diff
@@ -26,10 +26,10 @@ jobs:
         include:
           # yamllint disable rule:line-length
           - task: |
-              COMPARE_FEDORA_VERSIONS=./compare_fedora_versions MANIFEST_PATH=../../stratisd_proc_macros/Cargo.toml FEDORA_RELEASE=rawhide IGNORE_ARGS="--ignore-category low" make -f ../../Makefile check-fedora-versions
-              COMPARE_FEDORA_VERSIONS=./compare_fedora_versions MANIFEST_PATH=../../stratisd_proc_macros/Cargo.toml FEDORA_RELEASE=f35 IGNORE_ARGS="--ignore-category low" make -f ../../Makefile check-fedora-versions
-              COMPARE_FEDORA_VERSIONS=./compare_fedora_versions MANIFEST_PATH=../../Cargo.toml FEDORA_RELEASE=rawhide IGNORE_ARGS="--ignore-category low" make -f ../../Makefile check-fedora-versions
-              COMPARE_FEDORA_VERSIONS=./compare_fedora_versions MANIFEST_PATH=../../Cargo.toml FEDORA_RELEASE=f35 IGNORE_ARGS="--ignore-category low" make -f ../../Makefile check-fedora-versions
+              COMPARE_FEDORA_VERSIONS=./compare_fedora_versions MANIFEST_PATH=../../stratisd_proc_macros/Cargo.toml FEDORA_RELEASE=rawhide IGNORE_ARGS="--ignore-category low --ignore-high libcryptsetup-rs" make -f ../../Makefile check-fedora-versions
```
Comment (mulkieran) [inline .github/workflows/cargo.yml:29]: Drop the top two edits, these check stratisd_proc_macros.

File: Cargo.toml [hunk 2]
```diff
@@ -78,8 +78,10 @@ version = "0.1.0"
 path = "./stratisd_proc_macros"
 
 [dependencies.libcryptsetup-rs]
-version = "0.5.1"
+version = "0.6.0"
 features = ["mutex"]
+git = "https://github.com/jbaublitz/libcryptsetup-rs"
```
Comment (mulkieran) [inline Cargo.toml:83]: And drop these, of course.

**Review by mulkieran — CHANGES_REQUESTED** [review]
@jbaublitz Plz rebase and make the requested edits, and I'll merge this.

**Comment by mulkieran** [comment]
https://www.githubstatus.com/...

**Pull request merged**